dotnet/packages/Microsoft.Bot.Builder.M365/Microsoft.Bot.Builder.M365/AI/Action/DefaultActions.cs

```diff
@@ -90,15 +89,15 @@ public async Task<bool> SayCommand(ITurnContext turnContext, TState turnState, o
 
                 Attachment attachment = new() { Content = card, ContentType = AdaptiveCard.ContentType };
                 IMessageActivity activity = MessageFactory.Attachment(attachment);
-                await turnContext.SendActivityAsync(activity);
+                _ = await turnContext.SendActivityAsync(activity);
```

Comment:
I noticed that if you don't assign the expression a compiler warning will occur. Let's turn off warnings for the "expression not assigned to a variable" check.

Reply:
Removed.